gnosis/test/ApiTest.hex

```diff
@@ -1,4 +1,4 @@
 
-======= gnosis/test/ApiTest.sol:ApiTest =======
+======= ApiTest.sol:ApiTest =======
```

Comment:
specific reason to change this file path?

@denis-bogdanas what do you think?

Reply:
This is a generated file. Path is irrelevant.